remo/remozilla/tasks.py

```diff
@@ -27,7 +27,7 @@
              '&password={password}')
 URL = ('https://bugzilla.mozilla.org/rest/bug?token={token}'
        '&product=Mozilla%20Reps&component={component}&'
-       'include_fields={fields}&changed_after={timedelta}d&'
+       'include_fields={fields}&last_change_time={timedelta}&'
```

Comment:
I suggest we rename ``timedelta`` as well since this is not a ``timedelta`` but a ``timestamp``.